gp-translation-extended-api/gp-translation-extended-api.php

```diff
@@ -17,6 +17,57 @@ function translations_options_ok() {
 		$this->tmpl( 'status-ok' );
 	}
 
+	/**
+	 * Gets translation set by project and locale slug, and returns counts and
+	 * an array of untranslated strings (up to the number defined in GP::$translation->per_page)
+	 *
+	 */
+	function translation_get_untranslated_language() {
+		if ( ! $this->api ) {
+			$this->die_with_error( __( "Yer not 'spose ta be here." ), 403 );
+		}
+
+		$project_path          	= gp_get( 'project' );
+		$locale_slug           	= gp_get( 'locale_slug' );
+		$translation_set_slug  	= gp_get( 'translation_set_slug', 'default' );
+
+		if ( ! $project_path || ! $locale_slug || ! $translation_set_slug ) {
+			$this->die_with_404();
+		}
+
+		$filters = array(
```

Comment:
Just a suggestion, maybe integrate with gp-views to only fetch strings on the "Core Experience" view (which includes calypso)  and/or add 'client/' to the search term to includes only calypso strings (or do this via an additional endpoint query string).
This will make the strings to translate more relevant to the end user.

Reply:
@yoavf ah, good idea, thanks!! That's exactly what I need. 

I took a look at the gp-view plugin, and I think I need to grok it first. Do you mean port some of the functionality over, or cross-reference the plugin (assuming it's installed everywhere) ?

Reply:
cross-reference.
See for example here:
https://github.com/Automattic/gp-linguine/blob/87ba2267585dc754c5cf645c66683b35ea380fd4/gp-linguine.php#L96
Which fetches existing views, and then uses them in a form here:
https://github.com/Automattic/gp-linguine/blob/87ba2267585dc754c5cf645c66683b35ea380fd4/templates/send.php#L429

I think populating `filters[view]` should be enough.

Reply:
Awesome!

I'm using ?view={view_name}. `gp-view.php` [handles it](https://github.com/Automattic/gp-views/blob/master/gp-views.php#L51).

Thanks again @yoavf!!